timer/README.md

```diff
@@ -0,0 +1,9 @@
+## Steps to run this example
+
```

Comment:
just add some description to give the reader some idea of what the heck is going on?

```suggestion
# Timer Reminder Example

This example shows how to use a Timer to send a reminder email after some time elapses.

## Steps to run this example

```